tests/test_lab_serve.py

```diff
@@ -394,6 +400,15 @@ def test_vllm_args_config_with_ctx_tps(
     assert_tps(args, "4")
 
 
+def test_max_stable_vram_wait():
+    del os.environ["ILAB_MAX_STABLE_VRAM_WAIT"]
+    wait = get_max_stable_vram_wait(10)
+    assert wait == 10
```

Comment:
So if this fails, you leave default 30 seconds for the rest of tests. This is wrong, no? That's why you should really mock patch it.

Reply:
Switched to patch.  If it did fail, we would obviously notice, but patch is a much better solution.  Thanks!